src/_js/lib/Search.js

```diff
@@ -4,7 +4,7 @@ import Lunr from './Lunr';
 const renderResult = function(data) {
   const relativePath = data.path.replace(/^\/|\/$/g, '');
 
-  const url = `${window.location.origin}/${relativePath}`;
+  const url = `${window.location.origin}/${relativePath}/`;
```

Comment:
Why does this fix it?

Reply:
The sidebar determines what to show by matching the urls of the links to the current page. All the sidebar links end with a trailing slash.